hypervisor/hw/pci.c

```diff
@@ -449,6 +450,10 @@ static void pci_read_cap(struct pci_pdev *pdev)
 				for (idx = 0U; idx < len; idx++) {
 					pdev->msix.cap[idx] = (uint8_t)pci_pdev_read_cfg(pdev->bdf, offset + idx, 1U);
 				}
+			} else {
```

Comment:
had better add comment here to show it's for PCIE cap

Reply:
OK